frontend/public/components/monitoring/alerting.tsx

```diff
@@ -1093,7 +1095,7 @@ const SilencesDetailsPage = withFallback(
                     </dd>
                     <dt>{t('monitoring~Created by')}</dt>
                     <dd>{createdBy || '-'}</dd>
-                    <dt>{t('monitoring~Comments')}</dt>
+                    <dt>{t('monitoring~Comment')}</dt>
```

Comment:
I'm not sure the original, plural "Comments" wasn't intended. @kyoto?

Reply:
Yes. it is covered in #7386.